### DISCUSSION TITLE
Add Vercel Challenge bypass feature

### DISCUSSION THREAD

**Discussion by Sa7cez** [FlareSolverr/FlareSolverr#1345] [Ideas]
Please add new Vercel Challenge bypass feature. 

The first request from the site returns a 429 response with the cookie "x-vercel-challenge-token".

Then the js script and solver are loaded, which send the token and the response "x-vercel-challenge-solution".

After that, the server returns cookies that are linked to a specific browser fingerprint and allow further requests to be sent.

Site examples:
https://www.wishyouwereonchain.com/
https://mint.story.foundation/

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/1346
